### PR TITLE
Update all capability video links

### LINKED ISSUE #883: [Guide] Fix broken video link
## 🐛 Problem
Workload optimization video at the bottom of the page is broken.

## 👣 Repro steps
1. Go to https://microsoft.github.io/finops-toolkit/framework/capabilities/optimize/workloads
2. Scroll down to the embedded YouTube video

## 🤔 Expected
Should point to the correct video.

## ℹ️ Additional context
- Update the name of the page in the menu to "Workload optimization"
- Review other capability pages to ensure all other videos work as well

## 🙋‍♀️ Ask for the community

We could use your help:
1. Please vote this issue up (👍) to prioritize it.
2. Leave comments to help us solidify the vision.

### PR DESCRIPTION
**Pull request by flanakin**
## 🛠️ Description
Update all capability video links

Fixes #883 

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [x] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [x] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [ ] ❎ Docs not needed (small/internal change)

### CODE REVIEW

File: docs/_docs/framework/capabilities/manage/policy.md [hunk 1]
```diff
@@ -84,8 +84,8 @@ This capability is a part of the FinOps Framework by the FinOps Foundation, a no
 
 You can also find related videos on the FinOps Foundation YouTube channel:
 
-<!--[!VIDEO https://www.youtube.com/embed/{id}?list={list}]-->
-{% include video.html title="Cloud policy and governance videos" id="wxYMbhhA51dZNrky" list="PLUSCToibAswk2tx4PuWlztgSK9DZpL04b" %}
+<!--[!VIDEO https://www.youtube.com/embed/wiqCovCttOc?list=PLUSCToibAswnr2q_J-kn7Yii1CO2-PU35&pp=iAQB)]-->
```
Comment (arthurclares) [inline docs/_docs/framework/capabilities/manage/policy.md:87]: Didn't you want to add this video instead? https://www.youtube.com/embed/iL7H6oyzf7M?list=PLUSCToibAswk2tx4PuWlztgSK9DZpL04b&pp=iAQB

File: docs/_docs/framework/capabilities/optimize/architecting.md [hunk 2]
```diff
@@ -87,6 +87,11 @@ At this point, you should have a clear architectural guidance. As you move beyon
 
 This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Architecting for cloud](https://www.finops.org/framework/capabilities/architecting-for-cloud/) article in the FinOps Framework documentation.
 
+You can also find related videos on the FinOps Foundation YouTube channel:
+
+<!--[!VIDEO https://www.youtube.com/embed/zMonuvmZE1g&list=PLUSCToibAswm62kf2eILBPaRHobvxNy35&pp=iAQB]-->
```
Comment (arthurclares) [inline docs/_docs/framework/capabilities/optimize/architecting.md:92]: This link is broken. Is this what you tried to add? https://www.youtube.com/watch?v=E5OJmjrKYXk

File: docs/_docs/framework/capabilities/understand/reporting.md [hunk 3]
```diff
@@ -152,8 +152,8 @@ This capability is a part of the FinOps Framework by the FinOps Foundation, a no
 
 You can also find related videos on the FinOps Foundation YouTube channel:
 
-<!--[!VIDEO https://www.youtube.com/embed/{id}?list={list}]-->
-{% include video.html title="Reporting and analytics videos" id="CVTJLdcozj1eEpxT" list="PLUSCToibAswlDSQdehKhi7ysP2hmetigl" %}
+<!--[!VIDEO https://www.youtube.com/embed/pD9NeBOvspU?list=PLUSCToibAswlDSQdehKhi7ysP2hmetigl&pp=iAQB]-->
```
Comment (arthurclares) [inline docs/_docs/framework/capabilities/understand/reporting.md:155]: Maybe this one relates a bit more: https://www.youtube.com/watch?v=2qEcRzx26VA

**Review by arthurclares — CHANGES_REQUESTED** [review]
Requesting changes for one link that is broken and made suggestions for a couple others. Only one is broken, though.

File: docs/_docs/framework/capabilities/manage/policy.md (hunk 1 above)
Comment (flanakin) [reply docs/_docs/framework/capabilities/manage/policy.md:87]: That link would be for onboarding workloads. The one I have is for governance, so we should be good. I did add an extra parentheses, tho.
```suggestion
<!--[!VIDEO https://www.youtube.com/embed/wiqCovCttOc?list=PLUSCToibAswnr2q_J-kn7Yii1CO2-PU35&pp=iAQB]-->
```

File: docs/_docs/framework/capabilities/optimize/architecting.md (hunk 2 above)
Comment (flanakin) [reply docs/_docs/framework/capabilities/optimize/architecting.md:92]: I just confirmed. The link is correct. Looks like that video simply cannot be embedded 😕

File: docs/_docs/framework/capabilities/optimize/architecting.md [hunk 4]
```diff
@@ -87,6 +87,11 @@ At this point, you should have a clear architectural guidance. As you move beyon
 
 This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Architecting for cloud](https://www.finops.org/framework/capabilities/architecting-for-cloud/) article in the FinOps Framework documentation.
 
+You can also find related videos on the FinOps Foundation YouTube channel:
+
+<!--[!VIDEO https://www.youtube.com/embed/zMonuvmZE1g&list=PLUSCToibAswm62kf2eILBPaRHobvxNy35&pp=iAQB]-->
+{% include video.html title="Architecting for cloud videos" id="zMonuvmZE1g" list="PLUSCToibAswm62kf2eILBPaRHobvxNy35" %}
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/architecting.md:93]:
```suggestion
<!--
You can also find related videos on the FinOps Foundation YouTube channel:

<!- -[!VIDEO https://www.youtube.com/embed/zMonuvmZE1g&list=PLUSCToibAswm62kf2eILBPaRHobvxNy35&pp=iAQB]- ->
{% include video.html title="Architecting for cloud videos" id="zMonuvmZE1g" list="PLUSCToibAswm62kf2eILBPaRHobvxNy35" %}
-->
```

File: docs/_docs/framework/capabilities/understand/reporting.md (hunk 3 above)
Comment (flanakin) [reply docs/_docs/framework/capabilities/understand/reporting.md:155]: All of these links are to playlists for the respective capabilities. Ultimately, I'd like to link to playlists we own, but we can do that later.

**Review by arthurclares — APPROVED** [review]

**Pull request merged**